pkg/runtime/runtime_test.go

```diff
@@ -2123,8 +2135,7 @@ func TestInitBindings(t *testing.T) {
 		input.ObjectMeta.Name = "testinput"
 		input.Spec.Type = "bindings.testinput"
 		err := r.initBinding(input)
-		assert.Error(t, err)
```

Comment:
The error here was actually "app channel not initialized" which is not checked in this method anymore